soc/nxp/rw/firmwares.ld

```diff
@@ -3,6 +3,11 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
+#if defined(CONFIG_NXP_MONOLITHIC_WIFI)
+. = ALIGN(4);
+KEEP(*(.fw_cpu1))
```

Comment:
It would be nice to clarify:
- with a comment. What is this?
- why is it cpu1 (and not cpu0 or cpu2)?
- why is there no _wifi in its name?

Reply:
Hi @butok Place the wifi and ble firmwares in rodata section. A padding of 1 word is currently necessary, if both binaries are placed next to each other, the firmware loader
will detect the second binary as part of the first one, leading to
initialization issues.
for RW612, cpu1 is for wifi.
this name fw_cpu1, has been widely used in hal_nxp.

Reply:
> Hi @butok Place the wifi and ble firmwares in rodata section. A padding of 1 word is currently necessary, if both binaries are placed next to each other, the firmware loader will detect the second binary as part of the first one, leading to initialization issues. for RW612, cpu1 is for wifi. this name fw_cpu1, has been widely used in hal_nxp.

It's not clear for users:
- names should be self-descriptive. 
- comments should be added.

Reply:
Hi @butok add the comment, please review